src/styles/blocks/container.scss

```
@@ -0,0 +1,34 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(#{$min-block-width}, 1fr));
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+
+  @media (min-width: #{$min-block-width * $max-columns}) {
+    & {
+      grid-template-columns: repeat(#{$max-columns},
+      minmax(#{$min-block-width}, 1fr));
+    }
```

```suggestion
      grid-template-columns: repeat(#{$max-columns},
      minmax(#{$min-block-width}, 1fr));
```

`@media` is applied to the selector it is called in 😉